doc/developer-guide/developer-guide.rst

```diff
@@ -0,0 +1,98 @@
+.. _Developer_Guide:
+
+#########################################
+Electrometer Developer Guide
+#########################################
+
+The Electrometer is a standard salobj CSC.
+It communicates over a RS-232 serial communication.
+
+.. _Dependencies:
+
+Dependencies
+============
+
+* `SAL <https://ts-sal.lsst.io>`_ - v4.0.0
+* ts_salobj - v5.x.0
+* pyserial - v3.4.0
+
+.. Linking to the previous versions may also be worthwhile, depending on the CSC
+
+.. _API:
+
+Electrometer API
+=============================
+
+The content in this section is autogenerated from docstrings.
+
+.. The code below should insert the docstrings from the code.
+
+.. automodapi:: lsst.ts.electrometer
+    :no-main-docstr:
+    :no-inheritance-diagram:
+
+
+.. _Build:
+
+Build and Test
+==============
+
+.. prompt:: bash
+
+    setup -kr .
+    pip install pyserial
+    scons
+
+
+alternatively
+
+.. prompt:: bash
+
+    pip install .[dev]
+    pytest --cov lsst.ts.electrometer -ra
+
+
+.. _Usage:
+
+Usage
+=====
+
+.. prompt:: bash
+
+    RunElectrometer.py 1
+
+
+.. _Simulator:
+
+Simulator
+=========
+
+A prototype simulator is provided but it needs a way to convert a tcp/ip connection to serial.
+
+
+.. _Firmware:
+
+Updating Firmware of the Electrometer
+==================================================
+
+The firmware for the electrometer is not updatable.
+
+
+.. _Documentation:
+
+Building the Documentation
+==========================
+
+.. prompt:: bash
+
+    package-docs clean && package-docs build
+
+.. _Contributing:
+
+Contributing
+============
+
+Code and documentation contributions utilize pull-requests on github.
+Feature requests can be made by filing a Jira ticket with the `ts_electrometer2` label.
```

Comment:
ts_electrometer
... maybe consider grepping the repo for ts_electrometer2 to find all the instances.